x-pack/platform/plugins/shared/onechat/public/services/chat/chat_service.ts

```diff
@@ -38,7 +40,9 @@ export class ChatService {
       conversation_id: params.conversationId,
       connector_id: params.connectorId,
       capabilities: params.capabilities ?? getKibanaDefaultAgentCapabilities(),
-    };
+      attachments: params.attachments,
+    } as ChatRequestBodyPayload;
```

Comment:
the type casting shouldn't be necessary, the return type is already defined inline on L37